src/main/kotlin/demo/controllers/OrganizationController.kt

```diff
@@ -0,0 +1,49 @@
+package demo.controllers
+
+import demo.models.people.Organization
+import demo.services.OrganizationService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/api/organizations")
+class OrganizationController(@Autowired private val organizationService: OrganizationService) {
+    @GetMapping
+    fun getOrganizations(): ResponseEntity<MutableIterable<Organization>> {
+        return ResponseEntity(organizationService.getOrganizations(), HttpStatus.OK)
+    }
+
+    @GetMapping("/{organizationId}")
+    fun getOrganization(@PathVariable organizationId: String): ResponseEntity<Organization> {
+        val organization: Optional<Organization> = organizationService.getOrganization(organizationId)
+        return if (organization.isPresent)
+            ResponseEntity(organizationService.getOrganization(organizationId).get(), HttpStatus.OK)
+        else
+            ResponseEntity(null, HttpStatus.NOT_FOUND)
+    }
+
+    @PostMapping("/addorganization")
+    fun addOrganization(@RequestBody organization: Organization): ResponseEntity<Organization> {
+        val saved: Organization = organizationService.addOrganization(organization)
+
+        val httpHeaders = HttpHeaders()
+        httpHeaders.add(HttpHeaders.ACCEPT, "/api/organization/" + saved.id)
+        return ResponseEntity(saved, httpHeaders, HttpStatus.CREATED)
+    }
+
+    @PutMapping("update/{organizationId}")
```

Comment:
Głowy nie dam, ale nie musi być tutaj tego `update` przed `organizationId`. Spring rozróżni metody http i odwoła się do odpowiedniej funkcji - w tym wypadku i tak trzeba będzie zrobić request z put, bo nawet jak podasz tę ścieżkę, ale z postem, to nie zadziała.